db/migrate/20240726144332_add_contaminated_human_data_to_study.rb

```diff
@@ -0,0 +1,5 @@
+class AddContaminatedHumanDataToStudy < ActiveRecord::Migration[7.0]
+  def change
+    add_column :study, :contaminated_human_data_access_group, :string, default: nil
```

Comment:
We normally add migrations with functions `up` and `down`. `up` refers to adding the migration and `down` refers to reverting the migration back. So I think it's better if we adhere to this convention. You can see [`db/migrate/20240708081135_add_insert_size_to_aliquots.rb`](https://github.com/sanger/unified_warehouse/blob/7c812b497dc23f9c1b0c38175525e2411cbf2484/db/migrate/20240708081135_add_insert_size_to_aliquots.rb) for example.

Reply:
I am totally new to the Rails world, but from my reading, the up and down methods are considered the old style of migration in Rails. The change method is recommended because it can handle both applying and rolling back migrations. It is generally advised to use change instead of up and down.